ci/tools/Jenkinsfile.nix-cache

```diff
@@ -109,10 +109,16 @@ pipeline {
     stage('Upload') {
       steps { script {
         sshagent(credentials: ['nix-cache-ssh']) {
-          nix.shell("""
-              find /nix/store/ -mindepth 1 -maxdepth 1 -type d \
-                -not -name '*.links' -and -not -name '*-status-mobile-*' \
-                | xargs nix copy --to ${NIX_SSH_REMOTE}
+          nix.shell(
+            """
+              find /nix/store/ -mindepth 1 -maxdepth 1 -type d \\
+                -not -name "*.links" -and -not -name "*-status-mobile-*" \\
+                -print0 | xargs -0 nix-store -qR | sort -u > /tmp/store-paths.txt
+
+              nix-store --export < /tmp/store-paths.txt | \\
+                ssh ${env.NIX_SSHOPTS} ${params.NIX_CACHE_USER}@${params.NIX_CACHE_HOST} ${env.NIX_STORE_CMD} --import
```

Comment:
Stick to one command per shell, makes viewing what is broken in Jenkins logs easier.